src/components/Map/Legend.js

```diff
@@ -22,14 +21,9 @@ export function Legend(props) {
   );
 }
 
-export function LegendItem(props) {
-  const { title, color, condensed } = props;
-  return (
-    <LegendItemContainer condensed={condensed}>
-      <LegendItemHeader>
-        <ColorBox color={color} />
-        {title}
-      </LegendItemHeader>
-    </LegendItemContainer>
-  );
```

Comment:
The `LegendItemContainer` wrapper has margins, which we don't want on the `LocationMap` component, so I exported the `LegendItem` without the margin and using the wrapper to leave the legend as it was in other places.